prime-router/src/test/kotlin/ElementTests.kt

```diff
@@ -346,6 +346,14 @@ internal class ElementTests {
             "+1613-688-5335", // CA
             "+52 55 5080 2000", // MX
             "(230)7136595", // US, but invalid area code, but pass it through anyway
+            "213 555 5555", // US format
+            "+91(213) 555 5555 # 1234", // International (India) with extension number
+            "356-683-6541 x 1234", // US with extension number
+            "(985) 845-3258", // US format
+            "+91 (714) 726-1687 ext. 7923", // International (India) with extension
+            "(818) 265-7536 ext. 5264", // US with extension
+            "(874) 951-2157 # 8562", // US with extension
+            "+52 (213)478 9621 x 548", // MX with extension
```

Comment:
I can add more phone numbers with more variants like 555-555-5555x1234, let me know your thoughts

Reply:
Go ahead and add some examples where there isn't a space between the extension marker and the extension. Also try adding a really long extension as well as a short one. Also, these are all positive cases. Please add a few tests where there's an extension with no extension marker (e.g. `(717) 555-1212 0001`) and an extension marker with no number following (e.g. `(215) 555-1212 ext`.